### PR TITLE
RHDIP-2060: Update name reference

### PR DESCRIPTION
**Pull request by abrennan89**
**Versions:**
1.1+

**Jira link:**
https://issues.redhat.com/browse/RHIDP-2060

**Preview link:**
https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-196/admin-rhdh/#proc-install-rhdh-airgapped-environment-ocp-helm_admin-rhdh

**Reviews:**
- [x] QE: @zdrapela 
- [x] Peer review: @linfraze

### CODE REVIEW

**Review by zdrapela — CHANGES_REQUESTED** [review]
The `{ocp-brand-name}` is used only in prerequisites, as far as I know. Other parts of the documentation use generally the `{ocp-short}`.

File: installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
-To install {product-short} in an air-gapped environment, you must have access to the `registry.redhat.io` and the registry for the air-gapped environment. 
+To install {product-short} in an air-gapped environment, you must have access to the `registry.redhat.io` and the registry for the air-gapped environment.
 
 .Prerequisites
 
-* You have installed an {ocp-brand-name} ({ocp-short}) {ocp-version-min} or later.
+* You have installed an {ocp-brand-name} {ocp-version-min} or later.
 * You have access to the `registry.redhat.io`.
-* You have access to the {ocp-short} image registry of your cluster. For more information about exposing the {ocp-short} image registry, see https://docs.openshift.com/container-platform/{ocp-version}/registry/securing-exposing-registry.html[Exposing the registry] in OpenShift documentation.
+* You have access to the {ocp-brand-name} image registry of your cluster. For more information about exposing the image registry, see the {ocp-brand-name} documentation about https://docs.openshift.com/container-platform/{ocp-version}/registry/securing-exposing-registry.html[Exposing the registry].
```
Comment (nickboldt) [inline installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc:15]:
```suggestion
* You have access to the {ocp-short} image registry of your cluster. For more information about exposing the image registry, see the {ocp-short} documentation about https://docs.openshift.com/container-platform/{ocp-version}/registry/securing-exposing-registry.html[Exposing the registry].
```

File: installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
```
Comment (linfraze) [inline installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc:14]: Per minimalism principle 1, we typically use the _For more information about [topic], see [link]_ format (so the _in OpenShift documentation_ was never even necessary) - just an fyi; definitely won't block the merge

**Review by linfraze — APPROVED** [review]
Peer review complete - lgtm

**Review by zdrapela — APPROVED** [review]

**Pull request merged**